EIPS/eip-5659.md

```diff
@@ -0,0 +1,47 @@
+---
+eip: 5659
+title: Social Media URI Propagation Event
+description: A minimal primitive to emit URI events for social media.
+author: Tim Daubenschütz (@TimDaub), Auryn Macmillan (@auryn-macmillan)
+discussions-to: https://ethereum-magicians.org/t/social-media-uri-propagation-event/10893
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-09-15
+---
+
+## Abstract
+
+We introduce a minimal on-chain primitive to emit URIs using Ethereum's event log infrastructure.
+
+## Motivation
+
+The modern social network's base layer is distributing content behind Universal Resource Identifiers as defined by RFC 3986. Any web resource can be identified by an URI or a URL and so upgrading Ethereum with a standard way of distributing content is useful.
```

Comment:
You haven't convinced me that this EIP needs to exist. I think I'd like to see some potential use cases, both for publishers and for event consumers.

Reply:
ok, how would you frame this in the context of an EIP though? E.g. I feel like pointing out the countless problems of web2 content distribution could read too political

Reply:
I don't think you need to bash web2 to make a case for this EIP. What URIs do you expect to publish? Who do you expect to consume them? What applications are difficult to implement without this standard? These questions, answered in your Motivation section, would make for a more compelling proposal.